docs/das/daserver-instructions.md

```diff
@@ -1,36 +1,94 @@
 # Data Availability Server Instructions
 
-#### Image:
+## Description
+The Data Availability Server, `daserver`, allows storage and retrieval of transaction data batches for Arbitrum AnyTrust chains. It can be run in two modes: either committee member or mirror. Commitee members accept time-limited requests to store data batches from an Arbitrum AnyTrust sequencer, and if they store the data then they return a signed certificate promising to store that data. Commitee members and mirrors both respond to requests to retrieve the data batches. The data batches are addressed by a keccak256 hash of their contents.
```

Comment:
Should say explicitly what the role of a mirror is.

Reply:
Addressed.